client/my-sites/domains/domain-management/dataviews/domains-dataviews.tsx

```diff
@@ -15,30 +16,41 @@ import {
 	QueryParams,
 } from './query-params';
 import { useActions } from './use-actions';
+import useBulkActionNotice from './use-bulk-action-notice';
 import { useDomainsDataViewsContext } from './use-context';
 import { getDomainId } from './use-domains';
 import './style.scss';
 import { useFields } from './use-fields';
 import { initializeViewState, getFieldsByBreakpoint } from './view';
 
-type Props = {
+const connector = connect( null, { successNotice, errorNotice } );
```

Comment:
I think we should avoid using "obsolete" redux connector approach.
We should consider using hooks here. For example:

`const dispatch = useDispatch();`

and call it like: `dispatch( successNotice )`

Also, dispatching can be defined right inside the `useBulkActionNotice` hook.

Reply:
Hahah, turned out I commented the same thing :D https://github.com/Automattic/wp-calypso/pull/100576#pullrequestreview-2654077188